HW_1/task_3.py

```diff
@@ -19,3 +19,18 @@
 0
 Точка на отрицательной части оси абсцисс
 """
+print ("ВВедите координаты точки : ")
+x=float (input("х= "))
+y=float (input("y= "))
+if x > 0 and y > 0: 
```

Comment:
Условия подобного рода можно свернуть относительно нуля.
```suggestion
if x > 0 < y: 
```